new

arfathamit opened the pull request:
1. I added the law of motion for tax basis of capital stock (equation 9). I tried to add other features like the full version, but I have not made it work.

2. I changed the definition of a to z (total factor productivity) to make it consistent with the paper.

arfathamit committed:
new

jdebacker commented:
@arfathamit Thanks for this PR. Can you give a more descriptive title to the PR?

The changes from `a` to `z` are appreciated.

To confirm about the added complexity, are you saying that you could successfully solve the model with Equation 9 in it?  But then addition other equations broke the model?  If that't he case, what additional equation made the difference from modeling solving to not (I'm assuming you are adding one at a time)?  And what was the message you received indicating failure?

arfathamit commented:
@jdebacker I added equation 9 and k_tau as a variable like the full model file. I can still solve the model after adding this equation.

The HH budget constraint of the full model includes tau_c, tau_i, tau_k, tau_l, tau_d, and tau_ic, but our simple version does not. I added these taxes into the HH budget constraint, HH FOC, and Gov't budget constraint. When I did this, the program could not find the steady state.  I didn't add them at a time, but I will try this again.

arfathamit committed:
add the working file

arfathamit committed:
Delete Code_Working directory

arfathamit committed:
add the working file

arfathamit committed:
Merge branch 'main' of https://github.com/arfathamit/TaxMultipliersDSGE

arfathamit committed:
changing ss values
